changelog/v1.3.16/glooctl-warnings.yaml

```
@@ -0,0 +1,8 @@
+changelog:
+  - type: FIX
```

should do non user facing for docs changes

---

```suggestion
  - type: NON_USER_FACING
```
(understood that this was just moved from the other PR, would still like to change)